src/main/java/com/intuit/graphql/orchestrator/stitching/XtextStitcher.java

```diff
@@ -105,24 +107,30 @@ public RuntimeGraph stitch(List<ServiceProvider> serviceProviders) {
     //Stitch Graphs
     XtextGraph stitchedGraph = new XtextGraphFolder().fold(XtextGraph.emptyGraph(), xtextGraphMap.values());
 
+    //Service Metadata
+    final Map<String, ServiceMetadata> serviceMetadataMap = xtextGraphMap.values().parallelStream()
```

Comment:
I don't think we need parallelStream() here

Reply:
yeah, on second thoughts the task is too small to be executed in parallel. I will remove it.